docs/tutorials/02_introduction_legacy_runtime.ipynb

```diff
@@ -0,0 +1,544 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    service = IBMRuntimeService(auth="legacy")
</pre><p>Why do I need to explicitly do <code>auth</code>  here if the credentials are stored with this info?  Cant the service init and pick legacy if it is the only option?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
You don't if you only have 1 type (legacy or cloud) of account saved. I added a comment to clarify.